src/crypto/tls/delegated_credentials_test.go

```diff
@@ -293,10 +293,10 @@ func TestDelegateCredentialsValidate(t *testing.T) {
 	cert := dcTestCerts["dcP384"]
 	validTime := dcNow.Sub(cert.Leaf.NotBefore) + dcMaxTTL
 
-	delegatedCred, _, err := NewDelegatedCredential(cert, ECDSAWithP384AndSHA384, validTime, false)
+	delegatedCred, err := NewDelegatedCredential(cert, ECDSAWithP384AndSHA384, validTime, false)
 	if err != nil {
 		t.Fatal(err)
-	} else if delegatedCred == nil {
+	} else if delegatedCred.DC == nil {
```

Comment:
nits: `else` is not needed here.
`Fatal` includes a `FailNow`. See [godoc](https://golang.org/pkg/testing/#B.Fatal)